docs/release-notes/release-notes-0.2.0.md

```diff
@@ -0,0 +1,31 @@
+# About this release
+
+This is a maintenance release. It includes the following changes:
```

Comment:
A lot of stuff was added, this doesn't seem like a maintenance release

Reply:
What would you suggest? as there are various things changed I wouldn't give it a specific name. Maybe we should find a term for "catching up compatibility release"

Reply:
Every release will be a compatibility catch up release, so we don't have different kinds of releases, maybe formulate  it like "As any of the previous, this release includes changes that improve compatibility with epoch nodes, more specifically:"